.github/workflows/snyk-ecr-integration-apply.yml

```diff
@@ -0,0 +1,40 @@
+name:  snyk-ecr-integration apply terraform
```

Comment:
Let's rename the service to "snyk-integration" since we'll likely use the same IAM role for any other Snyk integrations we introduce in the future.